src/api/python/CMakeLists.txt

```
@@ -19,6 +19,36 @@ if(POLICY CMP0057)
   cmake_policy(SET CMP0057 NEW)
 endif()
 
+# Check that scikit-build is installed
+# Provides CMake files for Python bindings
+check_python_module("skbuild" "scikit-build")
+
+# setuptools for installing the module
+check_python_module("setuptools")
+
+# Find cmake modules distributed with scikit-build
+# They are distributed under <scikit-build directory>/resources/cmake
+execute_process(
+  COMMAND
+    ${PYTHON_EXECUTABLE} -c "from __future__ import print_function; \
+                             import os; import skbuild; \
+                             cmake_module_path=os.path.join(os.path.dirname(skbuild.__file__),
+                             'resources', 'cmake'); print(cmake_module_path, end='')"
```

Make sure that this is 80 chars wide, you can make a newline after -c, maybe something like this:
```suggestion
    ${PYTHON_EXECUTABLE} -c \
      "from __future__ import print_function; \
      import os; import skbuild; \
      cmake_module_path=os.path.join(os.path.dirname(skbuild.__file__),
      'resources', 'cmake'); print(cmake_module_path, end='')"
```

---

Good catch and thanks for the suggestion!